docs/project/glossary.md

```diff
@@ -43,7 +43,6 @@ terminology.
 | POGO | Profile Guided Optimization - see [details](https://blogs.msdn.microsoft.com/vcblog/2008/11/12/pogo/). |
 | ProjectN | Codename for the first version of [.NET Native for UWP](https://msdn.microsoft.com/vstudio/dotnetnative.aspx). |
 | R2R | Ready-to-Run. A flavor of native images - command line switch of [crossgen](../workflow/building/coreclr/crossgen.md). |
-| Redhawk | Codename for experimental minimal managed code runtime that evolved into [CoreRT](https://github.com/dotnet/corert/). |
```

Comment:
Maybe we want to keep it for historical reasons and just update the reference to NativeAOT?

We still have ton of `Rh` or `Rhp` prefixed methods that refer to it (please don't start cleaning that up until #115339 lands, it would leave to nasty conflicts).